### ISSUE TITLE
itemFromUUID() not working properly

### ISSUE THREAD

**Issue by OakLoaf** [Deanveloper/SkullCreator#24]
The function itemFromUUID() only works for players that have been on the server that you are getting the head on. Ideally this should not be a limitation

**Comment by deanveloper** [comment]
yea that kinda sucks

as a workaround, you can use any of the other methods, which should _all_ work without requiring the player to have previously been online

**Comment by deanveloper** [comment]
(ideally i will fix this in the future but i haven't touched this repo in years)

**Comment by OakLoaf** [comment]
Yeahh, I saw that it was quite an old repo haha. I kind of need to have the player head updating for my plugin, maybe I can use the URL method?

**Comment by deanveloper** [comment]
No, if you use the URL method it will not update. I'll see if I can fix it somehow lol (later)

**Comment by OakLoaf** [comment]
Ok ok, thank you haha

**Comment by CoolDCB** [comment]
Any news? :)

**Comment by OakLoaf** [comment]
Gonna bump this

**Comment by deanveloper** [comment]
Still no news, I don't really have an environment set up for MC developing/testing anymore, so it'd take quite a bit of time for me to solve this myself.

I've been thinking about archiving this repo but I'd like to find someone else to maintain if possible

**Comment by deanveloper** [comment]
For now the best solution will probably be to resolve the name for a UUID yourself and then pass that into the fromName methid
